gbn/queue.go

```diff
@@ -219,6 +468,45 @@ func (q *queue) processNACK(seq uint8) (bool, bool) {
 	return true, bumped
 }
 
+// proceedAfterTime will wait for the resendTimeout and then send an
+// awaitedACKSignal, if we're still awaiting the resend catch up.
+func (q *queue) proceedAfterTime(resendTimeout time.Duration) {
+	go func() {
```

Comment:
Should we instead call `go proceedAfterTime()` to make it more obvious at the call site that we're spinning up a goroutine?

Reply:
Good idea, thanks!